### DISCUSSION TITLE
Creating a subset based on orientation

### DISCUSSION THREAD

**Discussion by Emil-cmd** [mtex-toolbox/mtex#1396] [Ask Anything] [answered]
Hello. 

I am wondering if there is a quick way to create subset of grains within a certain orientation span? Lets say for example that I want to get a subset list of all grains at, or within, 10 degrees of the (110) orientation, for a cubic material? 
So lets say I create a vector 
v = vector3d(90,0,0); 
and want to find all grains that have an orientation up to a with of 10 degrees to this orientation. 

I am interested in subsetting these grains and plotting them in a grain map to see where they are located on my map to see if there is any local orientation preferensens. 

Thank you in advance.

**Comment by kilir** [comment, marked as answer]
Hi,
apart that `v` in your case is not an orientation you can subset for an orientation using something like
```
cond = angle(list_of_orientations, reference_orientation) < some_threshold_angle
subset_of_ori = list_of_orientations(cond)
```
Hope this helps,
Rüdiger
